sdk/storage/storage-file/package.json

```diff
@@ -17,8 +17,10 @@
     "node": ">=6.5.0"
   },
   "dependencies": {
-    "events": "3.0.0",
     "@azure/ms-rest-js": "1.2.6",
+    "@types/nock": "^9.3.1",
+    "events": "3.0.0",
+    "nock": "^10.0.6",
```

Comment:
"nock" and "@types/nock" don't belong here, should be added in the devDependencies!
Use "--save-dev" flag while doing npm install.

Reply:
Fixed! I'll keep that in mind for future changes.